migrations/V178__Add_MTO_Common_Milestone_And_Solution.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE mto_common_milestone (

```

Comment:
I think both `mto_common_milestone` and `mto_common_solution` should probably have 2 new fields on them:
- `name` (to represent what the common milestone/solution is called)
- `key` (to represent a predictable enum/identifier for the Milestone/Solution -- I've got enums for them in GQL here! https://github.com/CMS-Enterprise/mint-app/blob/aa5fe3e9570117c3db1fb42f9768da2a481a811d/pkg/graph/schema/types/model_to_operations_matrix.graphql#L116-L128)

Reply:
That makes sense to me! I'd be ok merging this to feature as is though, and implement the change in another PR for the sake of making other table relations. 

(I think just the migration will need to be re-numberd to 179 since category was merged.

Reply:
I'm good with that, then! @OddTomBrooks can you just fix the failing lint on this and push that back up and we should be good!

Reply:
I just pulled in the new branch -- you might need to pull it down and check migration #'s too!